src/test/java/io/vertx/core/http/HttpTest.java

```diff
@@ -2167,6 +2167,64 @@ public void testSendRangeFileFromClasspath() {
     await();
   }
 
+  @Test
+  public void testSendZeroRangeFileFromClasspath() {
+    server.requestHandler(res -> {
+      // hosts_config.txt is 23 bytes
```

Comment:
can you check this in the tests (hosts_config.txt is 23 bytes), someday we might modify hosts_config.txt and have this test not working correctly